x-pack/legacy/plugins/integrations_manager/public/constants.ts

```diff
@@ -4,41 +4,36 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import { IconType } from '@elastic/eui';
-import { AssetType, ServiceName } from '../common/types';
+import { KibanaAssetType, AssetType, ServiceName } from '../common/types';
+
+export const DisplayedAssets: Record<ServiceName, AssetType[]> = {
```

Comment:
```suggestion
// TODO: Add types so `kibana` can only have `KibanaAssetType`, `elasticsearch` only has `ElasticsearchAssetType`, etc
export const DisplayedAssets: Record<ServiceName, AssetType[]> = {
```

Reply:
I can have a type like this: 
`interface DisplayedAssets{
    kibana: KibanaAssetType[],
    elasticsearch: ElasticsearchAssetType[]
}`

But then the serve names aren't typed.  I'm not sure how to have it all, here.

Reply:
Right. I've got some ideas, but it's not important now. A comment to come back to it is good enough for me.